### PR TITLE
Added Linklist.cpp

### LINKED ISSUE #377: Linked List [C++]
#### This issue is about [Linked List](https://en.wikipedia.org/wiki/Linked_list).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by devesh-verma**
Fixes #377 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

**Comment by devshiva619** [comment]
@singhpratyush  I think I have marked correctly.

File: linked_list/LinkedList.cpp [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+        single_llist()
+        {
+            start = NULL;
+        }
+};
+
+/*
+ * Main :contains menu
+ */
+main()
```
Comment (Monal5031) [inline linked_list/LinkedList.cpp]: make this `int main()` and return `0`.

File: linked_list/LinkedList.cpp [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+        cout<<"3.Insert node at position"<<endl;
+        cout<<"4.Sort Link List"<<endl;
+        cout<<"5.Delete a Particular Node"<<endl;
+        cout<<"6.Update Node Value"<<endl;
+        cout<<"7.Search Element"<<endl;
+        cout<<"8.Display Linked List"<<endl;
+        cout<<"9.Reverse Linked List "<<endl;
+        cout<<"10.Exit "<<endl;
+        cout<<"Enter your choice : ";
+        cin>>choice;
```
Comment (Monal5031) [inline linked_list/LinkedList.cpp]: Hardcode all the things don't take user input, it is the cause of travis build failure.

File: linked_list/LinkedList.cpp [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+{
+    int info;
+    struct node *next;
+}*start;
+
+/*
+ * Class Declaration
+ */
+class single_llist
+{
```
Comment (Monal5031) [inline linked_list/LinkedList.cpp]: Please use standard cpp indentations throughout the code, and be consistent with the spacings.

**Comment by singhpratyush** [comment]
@devshiva619: Are you still on it?

**Pull request closed without merging**